cdap-standalone/src/main/resources/cdap-site.xml

```diff
@@ -164,6 +163,15 @@
     </description>
   </property>
 
+  <property>
+    <name>preview.enabled</name>
+    <value>true</value>
```

Comment:
checked with @sreevatsanraman, it will be disabled by default. will update that.